bundles/org.openhab.binding.bluetooth.generic/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.bluetooth.generic</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Generic Bluetooth Adapter</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.bluetooth</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.sputnikdev</groupId>
+      <artifactId>bluetooth-gatt-parser</artifactId>
+      <version>1.9.4</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>commons-collections</groupId>
+      <artifactId>commons-collections</artifactId>
+      <version>3.2.2</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Are these used anywhere?

Reply:
They aren't used in this binding, but they are transitive dependencies for the gatt-parser library and in order for them to be packaged into this binding's jar I have to explicitly label them as compile dependencies in the pom.

Reply:
Are you sure? In the GATT POM, only commons-beanutils is listed: https://github.com/sputnikdev/bluetooth-gatt-parser/blob/master/pom.xml

Reply:
I am sure, because without including it here karaf would throw a fit during compilation wanting me to add commons-collections to the feature.xml. I didn't want to do that so I added it as a compile dependency instead.

Reply:
I looked further into it and found that the commons-collections library is a dependency for the commons-beanutils library.